typewise-alert.cpp

```diff
@@ -1,71 +1,51 @@
 #include "typewise-alert.h"
 #include <stdio.h>
 
-BreachType inferBreach(double value, double lowerLimit, double upperLimit) {
-  if(value < lowerLimit) {
-    return TOO_LOW;
-  }
-  if(value > upperLimit) {
-    return TOO_HIGH;
-  }
-  return NORMAL;
+
+BreachType Maintenance_and_Support::inferBreach(double value, double lowerLimit, double upperLimit) {
+	if (value < lowerLimit) {
+		return TOO_LOW;
+	}
+	if (value > upperLimit) {
+		return TOO_HIGH;
+	}
+	return NORMAL;
 }
 
-BreachType classifyTemperatureBreach(
-    CoolingType coolingType, double temperatureInC) {
-  int lowerLimit = 0;
-  int upperLimit = 0;
-  switch(coolingType) {
-    case PASSIVE_COOLING:
-      lowerLimit = 0;
-      upperLimit = 35;
-      break;
-    case HI_ACTIVE_COOLING:
-      lowerLimit = 0;
-      upperLimit = 45;
-      break;
-    case MED_ACTIVE_COOLING:
-      lowerLimit = 0;
-      upperLimit = 40;
-      break;
-  }
-  return inferBreach(temperatureInC, lowerLimit, upperLimit);
+BreachType Maintenance_and_Support::classifyTemperatureAndPressureBreach(CoolingType coolingType, double temperatureInC) {
```

Comment:
Pressure not seen in this function? Do you want to introduce it later?
Refactor functions that have words like and,not,or,but in their names. They tend to collect more complexity

Reply:
i will update.